core/domain/suggestion_services.py

```diff
@@ -165,6 +165,22 @@ def _update_suggestion(suggestion):
     suggestion_model.put()
 
 
+def edit_suggestion(suggestion_id, new_change):
+    """Edits the submitted suggestion.
+
+    Args:
+        suggestion_id: obj. Id of suggestion to be changed.
+        new_change: dict(str, str). Dictionary contains the change in
+        suggestion.
+    """
+    suggestion = get_suggestion_by_id(suggestion_id)
+    change_cls = type(suggestion.change)
+    change_object = change_cls(new_change)
```

Comment:
Do this at the controller. That way if the dict is incorrect, the error will be thrown before it reaches this function itself. (Always try to throw errors early.)

Reply:
ok sir

Reply:
done